Libs/MRML/Widgets/Resources/UI/qMRMLSliceControllerWidget.ui

```diff
@@ -4,17 +4,26 @@
  <widget class="ctkPopupWidget" name="qMRMLSliceControllerWidget">
```

Comment:
However, It was not needed to remove the Reformat one in the ctkCombobox. It is automatically updated updateWidgetFromMRMLSliceNode. I will restore the original file